src/builtins.ts

```diff
@@ -340,6 +340,8 @@ export namespace BuiltinSymbols {
   export const i8x16_shr_u = "~lib/builtins/i8x16.shr_u";
   export const i8x16_any_true = "~lib/builtins/i8x16.any_true";
   export const i8x16_all_true = "~lib/builtins/i8x16.all_true";
+  export const i8x16_min = "~lib/builtins/i8x16.min";
```

Comment:
As these are not actual instructions I'd prefer leaving them out and only do `v128.min`

Reply:
ok, got it